rir/src/interpreter/interp.cpp

```diff
@@ -376,6 +375,7 @@ void warnSpecial(SEXP callee, SEXP call) {
     // TODO: turn this on?
     return;
 
+#if 0
```

Comment:
Why is this disabled? Can we get a comment so that this doesn't confuse us in the future?